src/table/virtualized-table/TotalsCell.tsx

```diff
@@ -31,7 +34,10 @@ const TotalsCell = ({ index, style, data }: TotalsCellProps) => {
         display: 'flex',
         alignItems: 'center',
         borderStyle: 'solid',
-        borderWidth: isLastColumn ? '0px' : '0px 1px 0px 0px',
+        borderLeftWidth: '0px',
```

Comment:
For some reason there seems to be a 3px border by default or inherited from somewhere. I couldn't see it anywhere in the devtools except under the layout tab

Reply:
Yeah I've noticed that as well, could be some default browser behaviour